src/main/resources/equation1.txt

```diff
@@ -0,0 +1,3 @@
+3 3 1 12
+2 5 7 33
+1 2 1 8
```

Comment:
Add input file to `.gitignore`

Reply:
boards are the part of the exercise so it can be placed on github as resource

Reply:
Board isn't resource. Resources are required to use program. For now you Putin one of many example, which can be solved by program in to JVM and require specific path for matrix. It is uregulowania online in IDE, not in general.